our_repo.py

```diff
@@ -80,7 +80,52 @@ def number_of_016_files(self):
 
         return elm_016_count
 
+    def get_files_for_017(self, pattern):
+        pattern = pattern.replace('.', '/')
+        all_files = glob.glob(f'{self.repo_dir}/**/{pattern}.elm', recursive=True)
+
+        return [ file for file in all_files
+            if self.what_kinda_file(file) == ELM_017_FILE
+        ]
+
+    def get_files_for_017_with_breakdown(self, pattern):
+        pattern = pattern.replace('.', '/')
+        all_files = glob.glob(f'{self.repo_dir}/**/{pattern}.elm', recursive=True)
+
+        return { file:self.how_trivial_to_port(file) for file in all_files
```

Comment:
:art: flake8 would complain about missing whitespace after ':' - thoughts on running it on CI? (I don't feel strongly about it, hmm but maybe it'll provide a good foundation for welcoming contributors)